### PR TITLE
udate README.md: add the instructions to run frontend in dev mode

### PR DESCRIPTION
**Pull request by Zaker237**
This pull request adds instructions to readme that guides a dev who want to start the frontend in dev mode.

### CODE REVIEW

**Review by Asam237 — CHANGES_REQUESTED** [review]
Add this,
To run the application in dev mode, do:
yarn dev or npm run dev

**Comment by Zaker237** [comment]
@Asam237  i added the yarn instructions as you requested

**Comment by Asam237** [comment]
In fact, it's a Nuxt JS project. First we had to use "nuxt build" to start Nuxt in development mode. However, to overcome this, we can directly use "npm run dev" or "yarn dev" 

Here is this error we get when we use "yarn start" or "npm run start" 
nuxtstart

**Review by Asam237 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -45,3 +45,12 @@ Then run the api:
 
 #### How to run the api with the DB
 `make start-api`
+
+#### Frontend
+
+To run the frontend you should jump to the ./frontend folder using the command cd ./frontend and run the following command:
+
+`npm run dev` or
+`yarn dev`
```
Comment (elhmn) [inline README.md:54]: we could have kept the `npm install` step as well. 
Also it is recommended to use npm as we don't have a `yarn.lock` file in the folder

**Review by elhmn — CHANGES_REQUESTED** [review]

**Review by elhmn — APPROVED** [review]

**Pull request merged**